src/components/cms/SkillCreator/SkillWizard.js

```diff
@@ -1030,6 +1060,38 @@ class SkillWizard extends Component {
             </a>
             <CenterDiv>
               <DropDownDiv>
+                <Popover
+                  open={open}
+                  anchorEl={anchorEl}
+                  onClose={this.handleClose}
+                  anchorOrigin={{
+                    vertical: 'bottom',
+                    horizontal: 'center',
+                  }}
+                  transformOrigin={{
+                    vertical: 'top',
+                    horizontal: 'center',
+                  }}
+                >
+                  <Text>
+                    Open Source Skills will show in the public directory and are
+                    editable by everyone just like a Wikipedia article. Private
+                    skills are only editable by the creator/owner of the skill.
+                    They are not listed in the public directory and can
+                    currently only be used by the creator/owner.
+                  </Text>
+                </Popover>
+                <DetailText>
+                  Open Source and Public Skill(Coming Soon) (
+                  <span
+                    onClick={this.handleClick}
+                    style={{ color: '#4285F4', cursor: 'pointer' }}
+                  >
+                    ?
+                  </span>
+                  ):
+                </DetailText>{' '}
+                <Switch color="primary" checked={true} disabled={true} />
```

Comment:
But, atleast make functionality to toggle. You have hardcoded checked to true.

Reply:
@akshatnitd Done